build.gradle

```diff
@@ -23,7 +23,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.2'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```

Comment:
I still get https://github.com/googlemaps/android-maps-utils/issues/800 when trying to run the demo app of the gms variant on Android Studio 4.1.2 w/ plugin 4.1.2. Does this work on a different Android Studio version?

Does this require plugin 4.1.2, or could we revert back to plugin 4.0.2 for now?

Reply:
The plugin does require 4.1.x of the Android Gradle Plugin and I'm also able to repro #800 using AS 4.1.2 although I could've sworn yesterday that I was able to get this to work. I also have the canary build installed so perhaps I was using that version of AS.

Reply:
Looks like the same issue occurs on the canary build of AS. I was able to get the demo app to run using the `installGmsDebug` task though